build.gradle

```diff
@@ -128,6 +131,9 @@ dependencies {
         compile project(':web-api-client')
     }
 
+    // Config for our code analytics lives in a centralized repo: https://github.com/MovingBlocks/TeraConfig
+    codeMetrics group: 'org.terasology.config', name: 'codemetrics', version: '1.3.2', ext: 'zip'
```

Comment:
Where does the version come from? I only see a 1.2.0 release on https://github.com/MovingBlocks/TeraConfig/releases.

And how does the `zip` bit work?

Reply:
Artifactory :-)

http://artifactory.terasology.org/artifactory/webapp/#/artifacts/browse/tree/General/libs-release-local/org/terasology/config/codemetrics/1.3.2

I don't remember how faithfully we tried to keep the GitHub releases section up to date with what actually made it to Artifactory. TeraConfig really needs a "build job" (or action) already. Our Gradle projects can only fetch from Artifactory though, so the releases on GitHub don't really matter at the moment.

`zip` bit tells Gradle to look for a zip file instead of a jar file (the default) when it resolves the dependency from Artifactory

Reply:
Up for some GitHub Actions magic? What does it take to build and release TeraConfig?